games-action/powermanga/powermanga-0.93.1.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A 2D arcade shoot-em-up with 41 levels and more than 2000 sprites"
```

Comment:
This is a bit too much information. How about just:

> A 2D arcade shoot'em up game.

Reply:
Got it!